includes/polling/storage/hpe.ilo.inc.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+if (!is_array($storage_cache1['ilo-storage'])) {
+    $storage_cache1['ilo-storage'] = snmpwalk_cache_oid($device, 'cpqHoFileSysEntry', null, 'CPQHOST-MIB', 'hpe-ilo');
+    echo("HPE ILO4 ");
+}
+
+$iind = 0;
+$storage_cache10 = array();
+
+echo($storage);
+
+foreach ($storage_cache1['ilo-storage'] as $index => $ventry) {
```

Comment:
I'd be interested to see the output of an snmpwalk for this bit, the code seems like it could be simplified but I'd need to see the data.